sdk/cosmosdb/cosmos/src/utils/hashing/v2.ts

```diff
@@ -0,0 +1,51 @@
+import { doubleToByteArrayJSBI } from "./encoding/number";
+import { BytePrefix } from "./encoding/prefix";
+// tslint:disable-next-line
```

Comment:
Same. Why is this needed?

Reply:
i had requires in there originally. just leftover, i'll remove